0004A30B0023308F.json

```diff
@@ -1,6 +1,6 @@
 {
   "gateway_conf": {
-    "gateway_ID": "0004A30B0023308F",
+    "gateway_ID": "B827EBFFFEA1DA7B",
```

Comment:
Something is odd here, the title of the pull request says exactly the opposite of what you did in the code change, ie. before it was matching and now the EUI in the file does not match the filename anymore.

Reply:
Hi,
The EUI is B827EBFFFEA1DA7B so I wanted to put this into the file and also have the filename match. Is this correct?
Rob.

Reply:
The issue is that the file is still called `0004A30B0023308F.json` (perhaps you haven't pushed the rename yet?)
Check it out:
![image](https://user-images.githubusercontent.com/933277/42136403-dc2c54b8-7d5a-11e8-81b6-62f39d42578a.png)